contrib/vexecutor/vexecutor.c

```diff
@@ -66,12 +75,47 @@ _PG_fini(void)
 	vmthd.ExecEndNode_Hook = NULL;
 }
 
-static PlanState* VExecInitNode(PlanState *node,EState *eState,int eflags)
+static void backportTupleDescriptor(PlanState* ps,TupleDesc td)
```

Comment:
What's the purpose of this function ? Would you give the comment ?

Reply:
Due to Vectorized type unrecognizable in normal execution node, the TupleDesc structure should backport metadata to normal type before we do V->N result pop up. At present, we only support scan node vectorized execution, so we force this backport function invoked. In the future, we will check if its parent node is "N" type. I will comment the function